### DISCUSSION TITLE
passing ref to SolitoImage with useRef

### DISCUSSION THREAD

**Discussion by sgrund14** [nandorojo/solito#439] [Q&A] [answered]
is there something special you have to do to pass a ref to the Image component from Solito?

I'm trying to do something simple like this

```
const imgRef = useRef();

return <SolitoImage ... ref={imgRef} />
```

but `imgRef.current` is always null, instead of containing the image element ref

**Comment by nandorojo** [comment]
Unfortunately Next doesn’t forward the ref to its image: https://github.com/vercel/next.js/discussions/43267

https://github.com/nandorojo/solito/blob/master/src/image/expo/image.web.tsx#L24C7-L30

**Reply by sgrund14** [reply]
hm, it's actually working for me when i switch to only using `Image` from next

```
import { Image } from 'next/image';
 
 ...
 
const imgRef = useRef();

return <Image ... ref={imgRef} />
```

like here, `imgRef.current` is defined and i can set style props and whatnot

**Reply by nandorojo** [reply]
Oh...interesting, maybe they silently fixed this then. Let's see.

**Reply by nandorojo** [reply, marked as answer]
Just released 4.1.2 which should address this: https://github.com/nandorojo/solito/releases/tag/v4.1.2

**Reply by sgrund14** [reply]
amazing ty
